doc/tutorials/using_grub.rst

```diff
@@ -0,0 +1,188 @@
+.. _using_grub:
+
+Using GRUB to boot ACRN
+#######################
+`GRUB <http://www.gnu.org/software/grub/>`_ is a multiboot boot loader
+used for many popular Linux distributions. It also supports booting the
+ACRN hypervisor.
+See `<http://www.gnu.org/software/grub/grub-download.html>`_
+to get the latest GRUB source code and `<https://www.gnu.org/software/grub/grub-documentation.html>`_
+for detailed documentation.
+
+The ACRN hypervisor can boot from `multiboot protocol <http://www.gnu.org/software/grub/manual/multiboot/multiboot.html>`_
+or `multiboot2 protocol <http://www.gnu.org/software/grub/manual/multiboot2/multiboot.html>`_.
+Comparing with multiboot protocol, the multiboot2 protocol adds UEFI support.
+
+The multiboot protocol is supported by the ACRN hypervisor by default.
+When the hypervisor is loaded by GRUB's ``multiboot`` command, the
+multiboot protocol is used, and the guest kernel or ramdisk must be
+loaded with the GRUB ``module`` command.
+
+The multiboot2 protocol is supported when ``CONFIG_MULTIBOOT2`` is
+enabled in Kconfig.  When the hypervisor is loaded by GRUB's
+``multiboot2`` command, the multiboot2 protocol is used, and the guest
+kernel or ramdisk must be loaded with GRUB ``module2`` command.
+
+The ACRN hypervisor binary is built with two formats: ``acrn.32.out`` in
```

Comment:
Can we somehow turn this into a bullet list. I think it would make it clearer and easier to read.